plugins/repository-azure/build.gradle

```diff
@@ -71,3 +71,24 @@ testClusters {
         keystore 'azure.client.integration_test.key', 'azure_key'
     }
 }
+
+String azureAccount = System.getenv("azure_storage_account")
```

Comment:
Admittedly, it's not so great that I duplicated the setup infrastructure from the nested qa project here but I didn't really see another straight forward way of getting this set up since I can't execute these tests from the nested project (don't have the class path set up with the S3 repository and such).
I'd suggest to (in a follow-up) simply flatten out the logic here the same we we have it for S3 where everything is executed from the plugin's top level Gradle project.